bootstrap.js

```diff
@@ -31,16 +41,17 @@ function install(data, reason) {}
 function startup(data, reason) {
   let uri = registerAddonResourceHandler(data);
 
-  let loaderModule =
-    Cu.import('resource://gre/modules/commonjs/toolkit/loader.js').Loader;
-  let { Loader, Require, Main } = loaderModule;
+  let loaderModule = getBaseLoader();
+  let { Loader, Require } = loaderModule;
   unload = loaderModule.unload;
 
   let loaderOptions = {
     paths: {
       "./": uri,
-      "": "resource://gre/modules/commonjs/"
+      // Remove when Firefox 56 is no longer supported.
+      "": "resource://gre/modules/commonjs/",
```

Comment:
Hmm, isn't this for using SDK modules only? Aren't they already removed? I would think this bit can already be removed...?

Reply:
Actually we still have a `module.exports = require("sdk/event/core");` for backward compatibility. 

But I will move the resource:// prefix to this require, it will be easier to get rid of pre FF57 code this way.